README.md

```diff
@@ -16,9 +16,16 @@ does not require an active daemon to run.
 
 ## Installation
 
+The easiest method is with the installation script:
+
+```
+$ curl -s https://raw.githubusercontent.com/SeerUK/tid/master/install.sh | sh
```

Comment:
Bear in mind, if you want to try this command out, you'll need to swap `master` to `installer`.